trl/commands/cli.py

```diff
@@ -60,9 +123,24 @@ def main():
             env=os.environ.copy(),
         )
     except (CalledProcessError, ChildProcessError) as exc:
-        console.log(f"TRL - {command_name.upper()} failed on ! See the logs above for further details.")
+        console.log("TRL - CHAT failed on ! See the logs above for further details.")
```

Comment:
Is this the intended message?

```suggestion
        console.log("TRL - CHAT failed! See the logs above for further details.")
```